exception_reports/decorators.py

```diff
@@ -41,10 +41,20 @@ def _exception_reports(func, *args, **kwargs):
             filename = gen_error_filename(extension='html')
             report_location = storage_backend.write(filename, data)
             setattr(e, 'report', report_location)
+
+            ExceptionType = type(e)
+
+            def my_str(self):
+                m = ExceptionType.__str__(self)
+                return f'{m} [report:{report_location}]'
+
+            NewExceptionType = type(ExceptionType.__name__ + "_Override", (ExceptionType,), {'__str__': my_str})
+            e.__class__ = NewExceptionType
```

Comment:
Hey this will mutate the original `e`, I'm not sure the consequence of that, but alternatively you could clone `e` and then mutate the `__class__` property.